test/e2e-tests.sh

```diff
@@ -78,6 +78,9 @@ if (( HTTPS )); then
   add_trap "turn_off_auto_tls" SIGKILL SIGTERM SIGQUIT
 fi
 
+# Enable allow-zero-initial-scale before running e2e tests (for test/e2e/initial_scale_test.go)
+add_trap "kubectl -n ${SYSTEM_NAMESPACE} patch configmap/config-autoscaler --type=merge --patch=\"{\"data\":{\"allow-zero-initial-scale\":\"true\"}}\"" SIGKILL SIGTERM SIGQUIT
```

Comment:
I think _this_ you want without trap — i.e. you want this now, not when the test finishes.